### ISSUE TITLE
accidentally quitting

### ISSUE THREAD

**Issue by burak-erkan** [Lawrence-Chiappelli/screendimmer#1]
I am using manjaro xfce
I click the system tray icon with right click but most of times accidentally closing the application. 

Can you add confirmation for quitting
Also swipe buttons Donate and Quit will be better for protecting accidentally closing as i do.

**Comment by Lawrence-Chiappelli** [comment]
> Can you add confirmation for quitting
> Also swipe buttons Donate and Quit will be better for protecting accidentally closing as i do.

That sounds like a good idea. I should get around to exploring those options soon, among other needed small quality of life adjustments.

**Comment by Lawrence-Chiappelli** [comment]
> Also swipe buttons Donate and Quit will be better for protecting accidentally closing as i do.

I instead added a label at the very bottom of the tray. This solves the issue of accidentally misclicking the bottom buttom from a micro mouse movement.

This was a simple oversight, being a top-taskbar user. So thanks for reaching out about that.

> Can you add confirmation for quitting

I moved the quit button to the center of the tray. It still doesn't prompt for user confirmation but I think that's okay given the convenience and low probability of accidentally clicking it.

Check out the rest of the changes I made [here](https://github.com/Lawrence-Chiappelli/screendimmer/releases/tag/0.4.0).
